src/ride_type.h

```diff
@@ -142,6 +142,8 @@ class RideInstance {
 	virtual void RemoveAllPeople() = 0;
 	bool CanBeVisited(const XYZPoint16 &vox, TileEdge edge) const;
 
+	virtual std::vector<XYZPoint16> get_all_positions() = 0;
+
```

Comment:
Just insert it in a single line between 142 and 143?
It's about removal as well.